.github/workflows/build.yml

```diff
@@ -0,0 +1,89 @@
+name: Build wheels and deploy
```

Comment:
```suggestion
name: Build wheels and deploy (to PyPI, if active)
```

Reply:
Don't really mind, but the "PyPI" name appears elsewhere in the admin-facing components already.